cclib/method/ddec.py

```diff
@@ -419,7 +474,7 @@ def condition_densities(self):
             self._candidates_bigPhi.append([bigphiAI[atomi]])
             self._candidates_phi.append([phiAI[atomi]])
 
-            fitphi = True  # when convergence is reached, this is modified to False.
+            phiconverged = False  # when convergence is reached, this is modified to True.
```

Comment:
Typically in Python we typically use snake case, so this could be `phi_converged`.